styles/main.css

```diff
@@ -345,4 +345,84 @@ main .discount-card {
 }
 .bottom-img{
   width: 250px;
+}
+main .hero-menu-cards .container .card img{
+  width: 100%;
+}
+main .hero-menu-cards .container .card .different-curry{
+  width: 100%;
+  height: 63%;
+}
+main .hero-menu-cards .container .card .different-pizza{
+  width: 100%;
+  height: 63%;
+}
+main .hero-menu-cards .container .card{
+  width: 350px;
+  margin: 0 auto;
+}
+
+@media only screen and (max-width:769px)
+{
+  main .hero-menu-cards .container {
+      margin-top: 4rem;
+      display: grid;
+      grid-template-columns: repeat(1, 1fr);
+      gap: 2rem;
+      text-align: center;
+    }
+    main .hero-menu-cards .container .card{
+      width: 100%;
+    }
+
+    main .hero-menu-cards .container .card img{
+      width: 70%;
+    }
+
+    main .hero-menu-cards .container .card .different-curry {
+      width: 70%;
+      height: 63%;
+    }
+
+    main .hero-menu-cards .container .card .different-pizza {
+      width: 70%;
+    }
+    
+    main .hero-menu-cards h2{
+      font-size: 2em;
+    }
+    main .hero-menu-cards .sub-title{
+      font-size: 1em;
+    }
+    main .hero-menu-cards{
+      padding-top: 1em;
+      padding-bottom: 1em;
+    }
+}
+
+@media only screen and (min-width:770px){
+  main .hero-menu-cards .container {
+      margin-top: 4rem;
+      display: grid;
+      grid-template-columns: repeat(2, 1fr);
+      gap: 2rem;
+      text-align: center;
+    }
+  main .hero-menu-cards .container .card .different-curry {
+      height: 67%;
+    }
+}
+
+@media only screen and (min-width:1201px) {
+  main .hero-menu-cards .container {
+    margin-top: 4rem;
+    display: grid;
+    grid-template-columns: repeat(3, 1fr);
+    gap: 2rem;
+    text-align: center;
+  }
+  main .hero-menu-cards .container .card .different-curry {
+    width: 70%;
```

Comment:
Remove this width. It is making the image distorted above 1200px screens
![image](https://user-images.githubusercontent.com/78860897/181739065-036839c9-338f-4800-b151-25f6f03cf647.png)